flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/CodeGenerator.scala

```diff
@@ -582,13 +562,14 @@ class CodeGenerator(
          |
          """.stripMargin
 
-    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping) + "\n"
+    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping, partialResults) + "\n"
```

Comment:
we do not need to pass `aggMapping, partialResults, fwdMapping, constantFlags, outputArity, mergeMapping` into genXXXX functions, as we can directly get them from the inputs of the generateAggregations(). Or you are intent to do this to make it easier to capture the dependence of each genXXXX functions?